compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/hoisting-nested-let-declaration-2.expect.md

```diff
@@ -36,7 +36,8 @@ function hoisting(cond) {
         items.push(bar());
       };
 
-      let bar = _temp;
+      let bar;
+      bar = _temp;
```

Comment:
We could rewrite these to be a `StoreLocal LValue: {kind: "Let"}` which would result in cleaner output. Note that we currently produce a `DeclareContext` followed by a `StoreContext` for all non-hoisted context variables